tests/page-objects/contacts/admin-user.wdio.page.js

```diff
@@ -0,0 +1,68 @@
+const addUserButton = () => $('a#add-user');
+const addUserDialog = () => $('h2=Add user');
+const userName = () => $('#edit-username');
+const userFullName = () => $('#fullname');
+const userRole = () => $('#role');
+const userPlace = () => $('//span[@aria-labelledby="select2-facilitySelect-container"]');
+const userAssociatedContact = () => $('//span[@aria-labelledby="select2-contactSelect-container"]');
+const userPassword = () => $('#edit-password');
+const userConfirmPassword = () => $('#edit-password-confirm');
+const saveUserButton = () => $('a=Add user');
+const logoutButton = () => $('span=Log out');
+const select2SearchInputBox = () => $('//input[@aria-controls="select2-facilitySelect-results"]');
+const select2Name = () => $('.name');
+const select2SearchContactInputBox = () => $('//input[@aria-controls="select2-contactSelect-results"]');
+const select2AssociatedContact = () => $('.name');
+
+const goToAdminUser = async () => {
+  await browser.url('/admin/#/users');
+};
+
+const openAddUserDialog = async () => {
+  await (await addUserButton()).waitForDisplayed();
+  await (await addUserButton()).click();
+  await (await addUserDialog()).waitForDisplayed();
+};
+
+const inputAddUserFields = async (username, fullname, role, place, associatedContact, password) => {
+  await (await userName()).addValue(username);
+  await (await userFullName()).addValue(fullname);
+  await (await userRole()).setValue(role);
+  await selectPlace(place);
+  await selectContact(associatedContact);
+  await (await userPassword()).addValue(password);
+  await (await userConfirmPassword()).addValue(password);
+};
+
+const selectPlace = async (place) => {
+  await (await userPlace()).click();
+  await (await select2SearchInputBox()).addValue(place);
+  await (await select2Name()).click();
+
+};
+
+const selectContact = async (associatedContact) => {
+  await (await userAssociatedContact()).click();
+  await (await select2SearchContactInputBox()).addValue(associatedContact);
+  await (await select2AssociatedContact()).click();
+
+};
+
+const saveUser = async ()  => {
+  await (await saveUserButton()).waitForDisplayed();
+  await (await saveUserButton()).click();
+  await (await addUserDialog()).waitForDisplayed({ reverse: true });
```

Comment:
ha, I expected there to be a `waitForNotDisplayed` but the reverse option is slick.